### PR TITLE
feat: custom project frontend

### PR DESCRIPTION
**Pull request by artikandri**
Task: 
- Added Custom as one of project types
- Implemented custom project page

What's changed: 
- frontend/components/project/FormCreate.vue: Added custom as one of available project types to choose when creating new project 
- frontend/domain/models/project/project.ts: added CustomProjectType mapping 
- frontend/i18n/*: added translation to "Custom Project" in 4 languages; translated with Google Translate

What's new: 
- frontend/pages/projects/_id/custom/index.vue: new page to handle annotation in custom project. Accessible through this link: http://localhost:3000/projects/_id/custom?page=_pageNumber 
- frontend/assets/images/tasks/custom.png: custom project thumbnail 

Notable: 
- the annotation page behaves strangely when changing the page (go to next sentence to annotate), if the annotation has relation and graphemeMode is enabled. This is solved by disabling graphemeMode on route change. 

Screenshots: 
image
image
image
image
image

### CODE REVIEW

File: frontend/pages/projects/_id/custom/index.vue [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    <template #content>
+      <v-card>
+        <v-card-title>
+          <label-group
+            :labels="categoryTypes"
+            :annotations="categories"
+            :single-label="exclusive"
+            @add="addCategory"
+            @remove="removeCategory"
+          />
```
Comment (teddy-f-47) [inline frontend/pages/projects/_id/custom/index.vue]: I think this is the code that handles the "categories" (the general label for a given text), is that correct? If so, I think the single-label props should be:
`:single-label="project.singleClassClassification"`

to follow the project's settings on creation.

Comment (artikandri) [reply frontend/pages/projects/_id/custom/index.vue]: Ah you're right XD  Updated.

**Review by teddy-f-47 — APPROVED** [review]
I think everything is ok now, thank you for the hard work! From my side, approved for merging.

**Comment by haanh764** [comment]
Thank you for your great work. I think everything is oke now. Approved for merging XD

**Review by haanh764 — APPROVED** [review]

**Pull request merged**